projects/ion/src/lib/alert/alert.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, Input, ViewChild, ElementRef, OnInit } from '@angular/core';
+import { StatusType } from '../core/types/status';
+import { IconType } from '../icon/icon.component';
+
+export interface IonAlertProps {
+  message: string;
+  type?: StatusType;
+  closable?: boolean;
+}
+
+@Component({
+  selector: 'ion-alert',
+  templateUrl: './alert.component.html',
+  styleUrls: ['./alert.component.scss'],
+})
+export class AlertComponent implements OnInit {
+  @Input() message!: string;
+  @Input() type?: StatusType = 'success';
+  @Input() closable? = false;
+  public iconType: IconType;
+  @ViewChild('ionAlert', { static: false }) private ionAlert: ElementRef;
+
+  closeEvent() {
+    this.ionAlert.nativeElement.remove();
+  }
+
+  setIcon() {
+    switch (this.type) {
```

Comment:
what do you think about using object literal?

Reply:
That change was related by Iury and Danilo too. i've already implemented the object literal